ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/baseActivity.component.ts

```diff
@@ -107,6 +108,7 @@ export abstract class BaseActivityComponent implements OnChanges, OnDestroy {
                     this.activityCode.emit(this.model.activityCode);
                     this.initSteps();
                 }
+                this.isLoaded$.next();
```

Comment:
Emit an event when a model is loaded.
It is used in ddp-sdk/../activity-redesigned.component.ts as to check whether the activity should use vertical progress indicator (_isVerticalProgress_ flag).